vignettes/RPyGeo.Rmd

```diff
@@ -18,13 +18,19 @@ knitr::opts_chunk$set(
 
 ## Introduction
 
-RPyGeo is a package providing access to ArcGIS geoprocessing tools in R. It archives this by building an interface between R and the ArcPy Python side-package via the reticulate package. 
-
-RPyGeo is not just a wrapper package for ArcPy. It aims to create an environment, in which the user can perform ArcGIS and R functions on the same dataset in a contiguous workflow. It should not be necessary to touch Python or ArcGIS. 
+**RPyGeo** establishes an interface to the geoprocessing tools of ArcMap in R.
+Since ArcMap only provides a Python API for a low-level access of its geoalgorithms, **RPyGeo** establishes a tunnel to Python via the **reticulate** package.
+<!-- what do you want to say with the next three sentences? -->
+RPyGeo is not just a wrapper package for ArcPy.
+It aims to create an environment, in which the user can perform ArcGIS and R functions on the same dataset in a contiguous workflow. 
+It should not be necessary to touch Python or ArcGIS. 
```

Comment:
I want to emphasize that RPyGeo is not just a remote control for ArcGIS. For example that you can directly load the output of a ArcGIS function with `rpygeo_load()` and apply a statistical model on it with R. But maybe this is unnecessary and everybody knows it?